src/modules/launcher/Wox.Infrastructure/Storage/StorageHelper.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Windows.Markup;
+
+namespace Wox.Infrastructure.Storage
+{
+    public class StorageHelper
+    {
+        // This detail is accessed by the storage items and is used to decide if the cache must be deleted or not
+        public bool clearCache = false;
+
+        
+        private String currentPowerToysVersion = String.Empty;
+        private String FilePath = String.Empty;
+
+        // As of now this information is not pertinent but may be in the future
+        // There may be cases when we want to delete only the .cache files and not the .json storage files
+        private enum StorageType
+        {
+            BINARY_STORAGE = 0,
+            JSON_STORAGE = 1
+        }
+
+        // To compare the version numbers
+        public static bool Lessthan(string version1, string version2)
+        {
+            string version = "v";
+            string period = ".";
+
+            string[] split1 = version1.Split( new string[] { version, period }, StringSplitOptions.RemoveEmptyEntries); 
+            string[] split2 = version2.Split( new string[] { version, period }, StringSplitOptions.RemoveEmptyEntries); 
+
+            for(int i=0; i<3; i++)
+            {
+                if(int.Parse(split1[i]) < int.Parse(split2[i]))
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        public string GetPreviousVersion()
+        {
+            GetFilePath();
+
+            if (File.Exists(FilePath))
```

Comment:
nit:  GetFilePath is called in a couple methods.  Can't this be implemented in the 'get' of of FilePath property?

Reply:
It is called only once now and it depends on the file that this storageHelper is associated with, so I am calling the function once in the constructor.